apps/aecontract/test/aect_test_utils.erl

```diff
@@ -62,9 +62,9 @@
 -include_lib("common_test/include/ct.hrl").
 -include_lib("aecontract/include/aecontract.hrl").
 -include_lib("aecontract/include/hard_forks.hrl").
--include_lib("aecontract/test/include/aect_sophia_vsn.hrl").
 -include_lib("aecontract/test/include/aect_contract_cache.hrl").
--include("../src/aect_sophia.hrl").
+-include("../../aecontract/test/include/aect_sophia_vsn.hrl").
```

Comment:
Isn't this a change in the wrong direction?

Reply:
You're right. Reverted; that one was a mistake.